data_discovery_ai/pipeline.py

```diff
@@ -19,8 +26,13 @@
 BASE_DIR = Path(__file__).resolve().parent
 
 # Construct the full path to the .ini file
-config_file_path = BASE_DIR / "common" / "keyword_classification_parameters.ini"
+config_file_path = BASE_DIR / "common" / KEYWORD_CONFIG
+if not config_file_path.exists():
+    raise FileNotFoundError(
+        f"The configuration file was not found at {config_file_path}"
+    )
```

Comment:
duplicate code appears in other file serving the same purpose, you should make this in one place, e.g make a config helper method

in that method , it will construct the file path and read the config details for eaxample.